go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/openshift/client-go
 
-go 1.19
+go 1.20
 
 require (
 	github.com/openshift/api v0.0.0-20230120195050-6ba31fa438f2
 	github.com/openshift/build-machinery-go v0.0.0-20220913142420-e25cf57ea46d
 	github.com/spf13/pflag v1.0.5
-	k8s.io/api v0.26.1
-	k8s.io/apimachinery v0.26.1
+	k8s.io/api v0.27.0-beta.0
+	k8s.io/apimachinery v0.27.0-beta.0
 	k8s.io/client-go v0.26.1
 	k8s.io/code-generator v0.26.1
```

Comment:
Those need to be bumped to `v0.27.0-beta.0` as well, everything that has `k8s.io/...` and ends with `0.26` is eligible for bump. All the rest should match what k8s has.